src/common-utils/install.sh

```diff
@@ -85,6 +85,7 @@ if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
         rsync \
         ca-certificates \
         unzip \
+        bzip2 \
```

Comment:
Can we bump the minor version here ( to 1.1.0 ) as we are adding a new package? (We follow semantic versioning) https://github.com/devcontainers/features/blob/main/src/common-utils/devcontainer-feature.json#L3

Features are released every time a PR is merged, these changes won't take effect unless the version specified here is different than the ones already published 😄

Reply:
👍🏽